scss/grouparoo.scss

```diff
@@ -311,6 +306,18 @@ a,
     padding: 0px;
   }
 }
+
+.customTable th {
```

Comment:
Being used in a few different places, so I've added it top level instead of to a module.